cypress/integration/404.spec.ts

```diff
@@ -0,0 +1,23 @@
+it('should display custom 404 page when not found', () => {
+  cy.visit('/not-existing-page', { failOnStatusCode: false })
+
+  cy.title().should('equal', '404 - Not Found | Octoclairvoyant')
+
+  cy.findByRole('heading', {
+    name: 'This page could not be found.',
+  }).should('exist')
+
```

Comment:
I'd like to have a test to check if the correct image is visible on the 404 page here. 😉

Reply:
Sure, I'll add that assertion later.